cli/CMakeLists.txt

```diff
@@ -31,6 +31,18 @@ configure_file (
   "${CMAKE_CURRENT_SOURCE_DIR}/CliConfig.h.in"
   "${CMAKE_CURRENT_BINARY_DIR}/CliConfig.h"
 )
+add_library(quickstep_cli_CommandExecutor CommandExecutor.cpp CommandExecutor.hpp)
+target_link_libraries(quickstep_cli_CommandExecutor
```

Comment:
We typically separate `add_library` and `target_link_libraries`.

So, move `target_link_libraries` after `# Link dependencies:`, and sort in alphabet order.